tests/integration/inventory

```diff
@@ -0,0 +1,2 @@
+[testgroup]
+testhost ansible_connection="local" ansible_pipelining="yes" ansible_python_interpreter="/Users/apajtas/miniconda3/bin/python"
```

Comment:
This local path probably shouldn't be there. Should it be `/usr/bin/python3`, or not even there if Ansible is able to cope?

Reply:
You're right, shouldn't be there 👍